server/src/main/java/fr/uga/l3miage/tp4/repositories/ExamRepository.java

```diff
@@ -0,0 +1,15 @@
+package fr.uga.l3miage.tp4.repositories;
+
+import fr.uga.l3miage.tp4.models.ExamEntity;
+import fr.uga.l3miage.tp4.models.SkillEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.Set;
+
+@Repository
+public interface ExamRepository extends JpaRepository<ExamEntity, Long> {
+
+    // Récupérer la liste des examens qui évaluent au moins une compétence en lien avec 'cardio' contenu dans le nom.(fonction + ExamComponent)
+    public Set<ExamEntity> findAllBySkillsContaining(String name);
```

Comment:
non dommage ici dommage vous voulez voir si le set de skills contients une entité particulière ^^' 

```suggestion
    public Set<ExamEntity> findAllBySkillsNameContaining(String name);
```